includes/degree-functions.php

```diff
@@ -300,13 +314,14 @@ function get_degree_description_modern_layout( $degree ) {
 	$modern_description_copy = get_field( 'modern_description_copy', $degree );
 	$modern_description_image = get_field( 'modern_description_image', $degree );
 	$modern_description_image_alt = get_field( 'modern_description_image_alt', $degree );
+	$modern_description_image_circle_class = get_field( 'modern_description_image_circle_class' ) ? 'rounded-circle' : '';
 
 	if( empty( $modern_description_heading ) && empty( $modern_description_copy ) ) return '';
 
 	ob_start();
 ?>
 	<section aria-label="Program description and highlights">
-		<div class="container py-lg-3 my-lg-5">
+		<div class="container py-lg-3 mt-4 my-lg-5">
```

Comment:
This change looks a little bit tight at the -sm and -md breakpoints; adding mt-sm-5 here would give the description a little bit of breathing room while still pushing the description up at -xs.